assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -679,6 +706,10 @@ def setup_and_validate_deepspeed(args: Namespace, do_validate: bool = True):
     # load deepspeed config
     ds_config_json = get_deepspeed_config_json(args)
 
+    # Update the deepspeed config based on the env flags
+    # NOTE cannot update args.deepspeed always as it could be an input to the port
```

Comment:
we can create a temp file right? and use it as it will be accessed only in finetune component?